OPEN_SOURCE_POLICY.md

```diff
@@ -0,0 +1,194 @@
+# Open Source Policy
+
+## 1. Introduction
+
+This Open Source Policy outlines the licensing, contribution, and compliance
+requirements for all code released under the Elixir project. By adhering to
+these guidelines, we ensure that our community, maintainers, and contributors
+uphold both legal and ethical standards while fostering a collaborative,
+transparent environment.
+
+This policy exists to support and protect the Elixir community. It aims to
+balance openness, collaboration, and respect for all contributors’ rights,
+ensuring that Elixir remains a trusted and innovative open source project.
+
+## 2. Scope
+
+This policy applies to the Elixir Programming language, located at
+https://github.com/elixir-lang/elixir. It covers every file, and contribution
+made, including documentation and any associated assets.
+
+<!--
+Goal:
+
+This policy applies to all projects in the `@elixir-lang` GitHub organization.
```

Comment:
This may be tricky and potentially not relevant for projects like tree-sitter-elixir, the website, etc. It may make sense to start from the opposite side and ask which projects require this. If compliance is really for the full organization, we may need to move some projects around and/or have a clearer rule which projects this applies to.

Reply:
That makes sense. I’ll think about it and propose a better solution.

Reply:
We can scope it to not be the whole organization. I propose to just remove the comment for now and only consider Elixir itself and later evaluate this part of the policy project by project.